.azure-pipelines/windows-steps.yml

```diff
@@ -21,7 +21,10 @@ steps:
   displayName: 'Display build info'
   condition: and(succeeded(), variables['testRunPlatform'])
 
-- script: PCbuild\rt.bat -q -uall -u-cpu -rwW --slowest --timeout=1200 -j0 --junit-xml="$(Build.BinariesDirectory)\test-results.xml" --tempdir="$(Build.BinariesDirectory)\test"
+- script: |
+    python.bat -m ensurepip --user
```

Comment:
Seems that this is happening but not actually installing `tzdata` in the environment correctly (I added a deliberately failing test to the TZ Data tests to check this and it continued to pass), but the GHA workflows are hitting these code paths, so we can defer getting the test requirements installed on AP until a later PR.